docker-compose.yml

```diff
@@ -84,6 +100,7 @@ services:
   # the correct image.
   #
   cassandra:
+    container_name: "node_cassandra"
```

Comment:
not sure you need to do this do you? when i use docker-compose up -d all the containers get the prefix of the directory i am in, e.g., ao8 or ao10. i'm thinking that you will use a different directory for ruby vs. node vs. whatever, so they would all be named differently regardless, right?

Reply:
I was under the impression that you don't really use docker containers. I like them named, it is more predictable. Do you need the folder name prepended, because you test with different versions?
My folder name is 'appoptics-apm-node', they also get a count appended. So it is `appoptics-apm-node_memcached_1` versus  `node_memcached` when named.
Not sure about -p, it is for the port when starting up with `run`.
Why should the mysql container name be removed?

Reply:
i use docker containers to run all the databases, i.e., everything other than my development environment. it's easy enough to change the name of the directory to `node` if you want that prefix. i just don't want it. and yes, it has a `_1` but that's never bothered me either.

docker-compose -p is different than docker -p: https://docs.docker.com/compose/reference/overview/

i'm not sure the interplay with `docker-compose run` and `docker-compose up -d` (the only command i use). it seems like `docker-compose up -d` will start a main container, so you'd be `docker exec -it ao8_main_1 /bin/bash` (or something like that) to connect to the already running container. or maybe you use `docker-compose run ...` without starting the other containers?

maybe it makes sense for you to add `-p node` if you want to change the naming? while i rarely have any conflicts i don't see the benefit in hard coding the name. you can use `docker-compose -p node run -p x:y /bin/bash` or something like that.

Reply:
Oh, and I'd remove the mysql container name just so it's consistent with the others.